tests/unit_tests/serialization.cpp

```diff
@@ -616,6 +616,50 @@ TEST(Serialization, serializes_ringct_types)
   ASSERT_EQ(bp0, bp1);
 }
 
+TEST(Serialization, key_encryption_transition)
+{
+  const cryptonote::network_type nettype = cryptonote::TESTNET;
+  tools::wallet2 w(nettype);
+  const boost::filesystem::path wallet_file = unit_test::data_dir / "wallet_9svHk1";
+  const boost::filesystem::path key_file = unit_test::data_dir / "wallet_9svHk1.keys";
+  const boost::filesystem::path new_wallet_file = unit_test::data_dir / "wallet_9svHk1_new";
+  const boost::filesystem::path new_key_file = unit_test::data_dir / "wallet_9svHk1_new.keys";
+  string password = "test";
+  bool r = false;
+  try
+  {
+    // Copy the old files before we overwrite them for this test
+    boost::filesystem::copy(wallet_file,new_wallet_file);
```

Comment:
The naming here confused me: As best as I can tell `_new` is a clone of the original data, so it's not really new so much as a backup.

Reply:
Fair enough, and easy to change.